docs/src/man/basics.md

```diff
@@ -0,0 +1,1220 @@
+                                                  4 columns and 985 rows omitted
+
+julia> empty!(german)
+0×10 DataFrame
+
+julia> german
+0×10 DataFrame
+```
+
+## Getting basic information about a DataFrame
+
+In this section we will learn about how to get basic information on our `German` DataFrame:
+
+The standard `size` function works to get dimensions of the DataFrame,
+```jldoctest dataframe
+julia> german = CSV.read((joinpath(dirname(pathof(DataFrames)),
+                                 "..", "docs", "src", "assets", "german.csv")),
+                       DataFrame) ;
+```
+
+```jldoctest dataframe
+julia> size(german), size(german, 1), size(german, 2)
+((1000, 10), 1000, 10)
+```
+
+To get the number of Rows or Columns in an AbstractDataFrame `german`, you can use `nrow()` and `ncol()`,
+```jldoctest dataframe
+julia> nrow(german), ncol(german)
+(1000, 10)
+```
+
+To get basic statistics of data in your DataFrame use `describe` (check out the help of describe for information on how to customize shown statistics).
+```jldoctest dataframe
+julia> describe(german)
+10×7 DataFrame
+ Row │ variable          mean     min       median  max              nmissing  ⋯
+     │ Symbol            Union…   Any       Union…  Any              Int64     ⋯
+─────┼──────────────────────────────────────────────────────────────────────────
+   1 │ id                499.5    0         499.5   999                     0  ⋯
+   2 │ Age               35.546   19        33.0    75                      0
+   3 │ Sex                        female            male                    0
+   4 │ Job               1.904    0         2.0     3                       0
+   5 │ Housing                    free              rent                    0  ⋯
+   6 │ Saving accounts            NA                rich                    0
+   7 │ Checking account           NA                rich                    0
+   8 │ Credit amount     3271.26  250       2319.5  18424                   0
+   9 │ Duration          20.903   4         18.0    72                      0  ⋯
+  10 │ Purpose                    business          vacation/others         0
+                                                                1 column omitted
+```
+
+To limit the columns shown by `desribe` use `cols` keyword argument:
+```jldoctest dataframe
+julia> describe(german, cols=1:3)
+3×7 DataFrame
+ Row │ variable  mean    min     median  max   nmissing  eltype
+     │ Symbol    Union…  Any     Union…  Any   Int64     DataType
+─────┼────────────────────────────────────────────────────────────
+   1 │ id        499.5   0       499.5   999          0  Int64
+   2 │ Age       35.546  19      33.0    75           0  Int64
+   3 │ Sex               female          male         0  String
+```
+
+You can adjust printing options by calling the `show` function manually: `show(german, allrows=true)`
+prints all rows even if they do not fit on screen and `show(german, allcols=true)` does the same for
```

Comment:
add an example with `allcols=true` and comment on a list of statistics computed by default

Reply:
Please help with `comment on a list of statistics computed by default`. 😞

Reply:
The default statistics reported are mean, min, median, max, number of missing values, and element type of the column. If columns contain `missing` values they are skipped when computing the summary statistics.